node/primitives/Cargo.toml

```
@@ -12,3 +12,4 @@ polkadot-statement-table = { path = "../../statement-table" }
 parity-scale-codec = { version = "1.3.5", default-features = false, features = ["derive"] }
 runtime_primitives = { package = "sp-runtime", git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
 sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" }
+schnorrkel = "0.9"
```

I have `Insert Final Newline` setting in my editor to avoid that:
```suggestion
schnorrkel = "0.9"

```